Sources/SKCore/BuildServerBuildSystem.swift

```diff
@@ -173,6 +173,15 @@ extension BuildServerBuildSystem: BuildSystem {
     return nil
   }
 
+  public func buildTargets(reply: @escaping ([BuildTarget]?) -> Void) {
```

Comment:
What's the reasoning for using optional instead of exposing the `Result` with an explicit error?

Reply:
The idea was that it is legitimate for a build system to not support the concept of targets, and so to return nil, but i'm fine with this being an error also.